string2.py

```diff
@@ -44,7 +51,11 @@ def not_bad(s):
 #  a-front + b-front + a-back + b-back
 def front_back(a, b):
     """Your code goes here.  Edit this docstring."""
-    return
+    # Using // rounds down, (5/2) -> 2.5 gets rounded to 2 
+    # To take advantage of that, I have made the numerator negative 
+    # (-5/2) -> -2.5 gets rounded down to -3, then -(-3) = 3, rounding up
+    # discovered this trick on a stack overflow post about rounding up when dividing
+    return(a[:-(-len(a) // 2)]+b[:-(-len(b) // 2)]+a[-(-len(a) // 2):]+b[-(-len(b) // 2):])
```

Comment:
A bit hard to read here -- there is a lot of len() // 2 going on.  Perhaps you can store (len(a) // 2) and (len(b) // 2) into some intermediate variables e.g. mid_a and mid_b?

Reply:
Okay great that would help, thanks!